src/vm/ceeload.cpp

```diff
@@ -74,6 +74,9 @@
 #include "peimagelayout.inl"
 #include "ildbsymlib.h"
 
+#ifdef FEATURE_PERFTRACING
```

Comment:
I'm surprised that you need this include.  What is being referenced from eventpipe here?

Reply:
This is left over from a previous evolution of this fix.
I will remove it.